derive/src/lib.rs

```diff
@@ -120,7 +120,20 @@
 //! 	}
 //! }
 //!
-//! # fn main() {}
+//! fn main() {
+//!	  let mut io = jsonrpc_core::MetaIoHandler::default();
+//!	  io.extend_with(RpcImpl::default().to_delegate());
+//!
+//!	  let server_builder = jsonrpc_tcp_server::ServerBuilder::with_meta_extractor(
+//!		io,
+//!		|request: &jsonrpc_tcp_server::RequestContext| Arc::new(Session::new(request.sender.clone()))
+//!	  );
+//!	  let server = server_builder
+//!		.start(&"127.0.0.1:3030".parse().unwrap())
+//!		.expect("Unable to start TCP server");
+//!
+//!	  // server.wait();
```

Comment:
"uncommented" because the example wouldn't terminate otherwise?

Reply:
Yes, is there some other way to prevent that? I guess we could mark it as "no-run"?

Reply:
I guess we could either mark it `no_run` or hide the "uncommented" line from the docs by `# // server.wait();`

Reply:
1. `no_run` feels inferior, since it won't detect the runtime errors
2. I think hidden lines are still being executed (they are just not displayed in the example)

I can remove the comment completely, but I think it's easier if we make users aware they have to wait for the server. Can add an explanation comment.

Reply:
> Can add an explanation comment.

Yes please. :)

Reply:
Added.